webpack.common.js

```diff
@@ -0,0 +1,81 @@
+const path = require("path");
+const MiniCssExtractPlugin = require('mini-css-extract-plugin')
+const CopyPlugin = require("copy-webpack-plugin")
+const BrowserSyncPlugin = require('browser-sync-webpack-plugin')
+const PACKAGE = require('./package.json')
+const theme = PACKAGE.theme
+
+module.exports = {
+  entry: {
+    screen: [`./src/Frontend/Themes/${theme}/src/Js/Index.js`, `./src/Frontend/Themes/${theme}/src/Layout/Sass/screen.scss`]
+  },
+  output: {
+    filename: 'bundle.js',
+    path: path.resolve(__dirname, `src/Frontend/Themes/${theme}/Core`),
+    clean: true,
+  },
+  mode: "development",
+  externals: ['jquery'],
+  watchOptions: {
+    aggregateTimeout: 600,
+    ignored: /node_modules/,
+  },
+  plugins: [
+    new MiniCssExtractPlugin({
+      filename: './Layout/Css/[name].css'
+    }),
+    new CopyPlugin({
+      patterns: [
+        {
+          from: "./Layout/Templates/**/*",
+          context: path.resolve(__dirname, "src", "Frontend", "Themes", theme, "src"),
+        },
+        {
+          from: "./Layout/Images/**/*",
+          context: path.resolve(__dirname, "src", "Frontend", "Themes", theme, "src"),
+        }
+      ],
+    }),
+    new BrowserSyncPlugin({
+      host: '127.0.0.1',
```

Comment:
Gaat dat altijd werken? Wat als ik op foo.bar.wip werk?

Reply:
De proxy kan mss wel een issue zijn. Daarvoor was encore wel gemakkelijker geweest. Is het een idee om die in een var te steken die je per project kan aanpassen, zodat daar de proxy in kan? Of kunnen we die ook in package.json steken bij opzet van het project?
Als dit het enige project is dat je hebt draaien gaat die symfony server altijd poort 8000 nemen, maar als je er twee hebt is het natuurlijk 8001.

Reply:
goh, maybe in comments iets bij zetten dan? Als het voor 90% gaat werken dan zou ik er geen tijd zijn steken.
Maar in theorie zouden we in sumo command wel iets kunnen doen dat dat aanpast naar de juiste url.